justfile

```diff
@@ -197,6 +197,11 @@ _dry-run-release: clean deps dart-build
 dry-run-release:
      @CI=true {{just_executable()}} _dry-run-release
 
+# Compiles the core lib for the given iOS target
+compile-ios target *args:
+    cd "$RUST_WORKSPACE"; \
+    cargo build --target {{target}} -p xayn-discovery-engine-core {{args}}
```

Comment:
This should target `bindings` not the core.

Reply:
> `just compile-ios aarch64-apple-ios` `just compile-ios aarch64-apple-ios --release`

In xayn_ai we are always compiling with `--release` because otherwise it is too slow to run. I think we should do the same here.

Reply:
done